<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
  background-color: &quot;orange&quot;;
  color: &quot;white&quot;;
}
QPushButton:hover {
  background-color:&quot;white&quot;;
  color: &quot;orange&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formulario">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_codigo">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CODIGO DEL CLIENTE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_cliente">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NOMBRE DEL CLIENTE</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_producto">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PPRODUCTO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_precio">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PRECIO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_cantidad">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CANTIDAD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ent_codigo">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ent_cliente">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ent_producto">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ent_precio">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="sbx_cantidad">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_botones">
     <item>
      <widget class="QPushButton" name="btn_calcular">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>CALCULAR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_limpiar">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>LIMPIAR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_salir">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>SALIR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light Condensed</family>
     </font>
    </property>
    <layout class="QFormLayout" name="resultados">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_importe">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IMPORTE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_descuentos">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>DESCUENTOS</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_interes">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>INTERÉS</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_total">
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TOTAL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="show_importe">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="show_descuento">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="show_interes">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="show_total">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light SemiCondensed</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gbox_interes">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>TIPO DE CRÉDITO INTERÉS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>81</width>
       <height>74</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layout_interes">
      <item>
       <widget class="QRadioButton" name="rbn_interes_0">
        <property name="font">
         <font>
          <family>Bahnschrift Light Condensed</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0 DÍAS 0%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbn_interes_15">
        <property name="font">
         <font>
          <family>Bahnschrift Light Condensed</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>15 DÍAS 8.5%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbn_interes_30">
        <property name="font">
         <font>
          <family>Bahnschrift Light Condensed</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>30 DÍAS 12.5%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbox_descuento">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>TIPO DE CLIENTE DESCUENTO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>91</width>
       <height>48</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layout_descuento">
      <item>
       <widget class="QRadioButton" name="rbn_mayorista">
        <property name="font">
         <font>
          <family>Bahnschrift Light Condensed</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MAYORISTA 4.5%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbn_minorista">
        <property name="font">
         <font>
          <family>Bahnschrift Light Condensed</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MINORISTA 1.9%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="text_display">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ayuda_en_python.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
